src/pages/studyView/charts/scatterPlot/StudyViewScatterPlot.tsx

```diff
@@ -197,39 +197,62 @@ export default class StudyViewScatterPlot extends React.Component<IStudyViewScat
     }
 
     @computed get data() {
-        return getDownsampledData(this.props.data, this.props.sampleToAnalysisGroup, this.dataSpaceToPixelSpace);
+        return getDownsampledData(this.props.data, this.dataSpaceToPixelSpace, this.props.sampleToAnalysisGroup);
     }
 
     @computed get scatters() {
         // sort NA to the beginning - it should be rendered at the bottom
         // otherwise, order doesnt matter. here we'll just use the same order as given in the prop, bc _.sortBy is stable sort
+        console.log("------------");
```

Comment:
I think all code changes related to StudyViewScatterPlot are no longer needed.

Reply:
its necessary for compatibility, and theres some small changes here and there, I think just lets let these changes through